jenkins/build.sh

```diff
@@ -5,14 +5,12 @@ set -e
 HOST=$(hostname)
 
 if [[ "$HOST" == kesch* || "$HOST" == escha* ]]; then
-    module load craype-network-infiniband
+    module load cmake/3.13.4
```

Comment:
Do we still test CMake 3.12.4 somewhere? Otherwise we might want to change `cmake_minimum_required` to make sure that we test what we expect.